RefRed/configuration/loading_configuration.py

```diff
@@ -79,8 +79,24 @@ def load_config_in_big_table_data(self):
         self.populate_main_gui_general_settings()
 
     def populate_big_table_data_with_lconfig(self):
-        dom = self.dom
-        RefLData = dom.getElementsByTagName('RefLData')
+        # Get the version of the software and warn user if they are trying
+        # to load an older xml file.
+        version_tag = self.dom.getElementsByTagName('version')
```

Comment:
This version tag should be _RefRed_'s version number, not _LiquidReflectometer_'s version number. Am I right?

Reply:
I think this should be the reduction version, since data can be reduced outside of refred. We may also have no change to refred but still have a change to the reduction. So I can see us wanting to track that in the future.

That said, I found that the xml that is exported by refred doesn't contain the same header info as the reduction. I fixed that and added more info per your other comment below.